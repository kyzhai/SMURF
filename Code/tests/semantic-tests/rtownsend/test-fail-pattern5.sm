/* All variables in pattern list must be unique */

f a:rest [[[a]]] = 56

main = []
