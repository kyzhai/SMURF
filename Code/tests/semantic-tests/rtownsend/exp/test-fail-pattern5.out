Fatal error: exception Sast.Multiple_patterns("a")
